torch/optim/rmsprop.py

```diff
@@ -11,6 +11,9 @@ class RMSprop(Optimizer):
     The centered version first appears in `Generating Sequences
     With Recurrent Neural Networks <https://arxiv.org/pdf/1308.0850v5.pdf>`_.
 
+    Note that the implementation here takes the square root of the gradient average before
+    adding epsilon.
```

Comment:
nits: It would be nice to have an equation like one of those that you showed me offline; equations are easier to read sometimes. Also, it might be good to document that this differs from the TF implementation specifically because I'm not sure if someone using RMSprop knows exactly where TF does the square root without investigation.

Reply:
Good points. I've added a note about tensorflow and the effective learning rate for RMSProp as `\alpha/(\sqrt{v} + \epsilon)`.